management/backup.py

```diff
@@ -222,6 +222,7 @@ def perform_backup(full_backup):
 	# Stop services.
 	shell('check_call', ["/usr/sbin/service", "dovecot", "stop"])
 	shell('check_call', ["/usr/sbin/service", "postfix", "stop"])
+	shell('check_call', ["/usr/sbin/service", "php5-fpm", "stop"])
```

Comment:
This is just nitpicking as I imagine that in reality these all stop so quickly it won't matter. But just in case one took a while to stop, I think a better order to stop them in would be:
1. php-fpm first (because z-push depends on Postfix and Dovecot)
2. postfix second (because it depends on Dovecot for authentication)
3. dovecot last (because it depends on nothing)